src/sql/workbench/services/connection/browser/passwordChangeDialog.ts

```diff
@@ -119,7 +117,7 @@ export class PasswordChangeDialog extends Modal {
 		this._okButton.enabled = false;
 		this._cancelButton.enabled = false;
 		this.spinner = true;
-		this.changePasswordFunction(this._profile, this._params, this._uri, this._passwordValueText.value, this._confirmValueText.value).then(
+		this.changePasswordFunction(this._profile, this._uri, this._passwordValueText.value, this._confirmValueText.value).then(
 			() => {
```

Comment:
1. use await + try/catch to improve the readability of the code
2. also need to handle the error scenario.